changelog/db.changelog-1.4.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+<changeSet id ="1" author="Leigh Godson">
+    <tagDatabase tag="v1.4.0" />
+</changeSet>
+<changeSet id="2" author="Leigh Godson">
+        <createTable tableName="dax">
+            <column name="paymentReference" type="VARCHAR(30)">
+                <constraints primaryKey="true" nullable="false" primaryKeyName="daxId_pkey"/>
+            </column>
+            <column name="calculationId" type="INTEGER"/>
+            <column name="paymentPeriod" type="VARCHAR(200)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="totalQuarterlyPayment" type="NUMERIC">
+                <constraints nullable="false"/>
+            </column>
+            <column name="transDate" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="datePublished" type="DATE">
+                <constraints nullable="true"/>
```

Comment:
`<constraints nullable="true"/>` not necessary

Reply:
removed as suggested